lib/elixir/lib/keyword.ex

```diff
@@ -571,11 +571,23 @@ defmodule Keyword do
       iex> Keyword.merge([a: 1, b: 2], [a: 3, d: 4, a: 5])
       [b: 2, a: 3, d: 4, a: 5]
 
+      iex> Keyword.merge [a: 1], [2, 3]
+      ** (ArgumentError) expected a keyword-list in second argument, got: [2, 3]
```

Comment:
Also, I think it reads better as "expected a keyword list as the second argument, got: ...".